### PR TITLE
fix dashboard scheme. fix linter issue

### PR DESCRIPTION
**Pull request by mukundansundar**
Signed-off-by: Mukundan Sundararajan <65565396+mukundansundar@users.noreply.github.com>

# Description

Fixes dashboard scheme which was missed when changing it to `net.JoinHostPort`
Also fixed a linter issue that popped up.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1091 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: cmd/dashboard.go [hunk 1]
```diff
@@ -165,7 +165,7 @@ dapr dashboard -k -p 0
 			}()
 
 			// url for dashboard after port forwarding.
-			var webURL string = net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort))
+			var webURL string = fmt.Sprintf("%s%s", "http://", net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort)))
```
Comment (mukundansundar) [inline cmd/dashboard.go:168]: dashboardHost only accepts IP or `localhost` . That is already validated before in a `utils.IsAddressLegal` call

Comment (shubham1172) [inline cmd/dashboard.go:168]:
```suggestion
			var webURL string = fmt.Sprintf("http://%s", net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort)))
```

Comment (shubham1172) [reply cmd/dashboard.go:168]: On another thought, is there any utility for using `net.JoinHostPort`?

Comment (shubham1172) [reply cmd/dashboard.go]: @mukundansundar ?

Comment (mukundansundar) [reply cmd/dashboard.go]: I have not found a utility as yet ... IMO that can be a refactor that can be done separately ...

**Review by DeepanshuA — APPROVED** [review]
LGTM

**Pull request merged**